wagtail/admin/templates/wagtailadmin/home/upgrade_notification.html

```diff
@@ -1,17 +1,17 @@
 {% load i18n wagtailcore_tags wagtailadmin_tags %}
 {% wagtail_version as current_version %}
 <div
-    class="w-panel-upgrade-notification panel nice-padding"
-    data-upgrade-notification
-    data-current-version="{{ current_version }}"
-    {% if lts_only %}data-upgrade-lts-only{% endif %}
-    style="display:none"
+    class="w-panel-upgrade panel nice-padding w-hidden"
+    data-controller="w-upgrade"
+    data-w-upgrade-current-version-value="{{ current_version }}"
+    data-w-upgrade-lts-only-value="{{ lts_only|yesno:'true,false' }}"
```

Comment:
Note here - this new approach is always rendering `data-w-upgrade-lts-only` but the tests are expecting this to only show sometimes. 

An alternative approach would be to something like this. 

```suggestion
    {% if lts_only %}data-w-upgrade-lts-only-value="true"{% endif %}
```